pages-functions-cors/wrangler.toml

```diff
@@ -0,0 +1,3 @@
+name = "" # todo
```

Comment:
Pages doesn't read from `wrangler.toml` at the moment. Can you remove this to avoid confusion?